### ISSUE TITLE
Conda build workflow fails because of branch name has `/` or `-`

### ISSUE THREAD

**Issue by qin-yu** [hci-unihd/plant-seg#219]
I doubt if this is the correct. In other repos `initials/the-purpose` is a common practice.

Error:

```
Run conda install -q conda-build
  conda install -q conda-build
  conda build -c conda-forge conda-recipe
  shell: /usr/bin/bash -l {0}
  env:
    RELEASE_VERSION: qy/fix-halo-padding
    INPUT_RUN_POST: true
    CONDA_PKGS_DIR: /home/runner/conda_pkgs_dir
Channels:
 - local
 - conda-forge
 - defaults
Platform: linux-6[4](https://github.com/hci-unihd/plant-seg/actions/runs/8653881833/job/23730010647#step:7:4)
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done

...
...

Error: bad character '/' in package/version: qy/fix-halo-padding
Error: Process completed with exit code 1.
```

I believe this is caused by using branch name as version:

```
Run echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV

Run echo $RELEASE_VERSION
qy/fix-halo-padding
```

**Comment by qin-yu** [comment]
Use timestamp as release version fixes the problem:
https://github.com/hci-unihd/plant-seg/actions/runs/8654775868/job/23732604869

**Comment by qin-yu** [comment]
The fix 04a247235892459037b7e64ea70585bdb030acc7 made the version on `lcerrone` conda channel to be a timestamp.

![image](https://github.com/hci-unihd/plant-seg/assets/17722010/e0f7f910-c2c1-4f22-96f5-ab6d2a522193)

**Comment by qin-yu** [comment]
![image](https://github.com/hci-unihd/plant-seg/assets/17722010/a57a4bde-b9d7-48cb-8a5f-4401569a91f0)
